examples/images_contours_and_fields/plot_streamplot.py

```diff
@@ -22,30 +22,30 @@
 V = 1 + X - Y**2
 speed = np.sqrt(U**2 + V**2)
 
-fig = plt.figure(figsize=(7, 9))
-gs = gridspec.GridSpec(nrows=3, ncols=2, height_ratios=[1, 1, 2])
+
+gs = {'height_ratios': [1, 1, 2]}
+fig, axes = plt.subplots(nrows=3, ncols=2, gridspec_kw=gs, figsize=(7, 9))
+((ax0, ax1), (ax2, ax3), (ax4, ax5)) = axes
```

Comment:
There is really no need to unpack these, so don't do so.  Just refer to `axs[0, 1]` etc.